test/unit/test_logger.py

```diff
@@ -126,3 +126,28 @@ def test_cyan_text():
     x = '{}{}{}'.format(colorama.Fore.CYAN, 'foo', colorama.Style.RESET_ALL)
 
     assert x == logger.cyan_text('foo')
+
+
+def test_markup_detection_pycolors_0(monkeypatch):
+    monkeypatch.setenv('PY_COLORS', '0')
+    assert logger.should_do_markup() is False
```

Comment:
Comparing expressions which are already boolean with boolean literals is unpythonic. Here's how to do it:
```suggestion
    assert not logger.should_do_markup()
```